ui-manchette/src/components/OperationalPoint.tsx

```diff
@@ -32,9 +32,9 @@ const OperationalPoint = ({
     >
       <div className="op-position justify-self-start text-end">{positionMmToKm(position)}</div>
```

Comment:
class should be renamed too

```suggestion
      <div className="waypoint-position justify-self-start text-end">{positionMmToKm(position)}</div>
```